tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -1119,6 +1119,7 @@ void test() {
 
 #if !defined(_DLL) || _ITERATOR_DEBUG_LEVEL == DEFAULT_IDL_SETTING
     test_locale<wchar_t>();
+    test_locale<char>();

```

Comment:
should we be running these if the literal encoding is _not_ utf-8?

Reply:
The test is only compiled with `/utf-8`.

https://github.com/microsoft/STL/blob/ba73f5a60a7a650df0f9d54f9710c33bb1c0d0b2/tests/std/tests/P0355R7_calendars_and_time_zones_formatting/env.lst#L4-L6

We can test that, if the literal encoding is not utf-8, `format(locale{"zh-CN"}, STR("{:^22L%Y %B %d %A}"), 2021y / June / 16d)` produces `" 2021 \xC1\xF9\xD4\xC2 16 \xD0\xC7\xC6\xDA\xB6\xFE  "` (text is GB2312-encoded) after `setlocale(LC_ALL, ".936")` and produces `" 2021 \xE5\x85\xAD\xE6\x9C\x88 16 \xE6\x98\x9F\xE6\x9C\x9F\xE4\xBA\x8C  "` (text is UTF8-encoded) after `setlocale(LC_ALL, ".UTF-8")`, but I feel that it isn't worth the trouble.

Reply:
It would be nice if this was checked for C++17 as well.

Reply:
Chrono formatting is C++20-only.